bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/tracer/TracersStore.java

```diff
@@ -15,12 +15,11 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
```

Comment:
Saw the other classes are maintaining this new line.
I think we should add the new line to be consistent.

Reply:
Actually, most of the classes don't have a new line between header and package declaration. I removed some new lines to be consistent.